app/models/spaCy/spacy.py

```diff
@@ -6,8 +6,8 @@ def __init__(self):
     return None
 
   def load(self):
-    self.nlp = spacy.load("./app/models/spaCy")
+    self._nlp = spacy.load("./app/models/spaCy")
```

Comment:
What's the difference between `self.nlp` and `self._nlp`?

Reply:
the leading underscore is just naming convention that says "this is a private variable." There aren't actually public/private variables in python, so it doesn't make a real difference. I'm just trying to be PEP friendly